docs/learn/output_parsers.md

```diff
@@ -36,17 +53,53 @@ print(f"Title: {book.title}")
 print(f"Author: {book.author}")
 ```
 
-In this example, the `parse_book_recommendation` function serves as an Output Parser, transforming the raw response into a structured `Book` instance.
+In this example:
+
+1. We define a `Book` model using Pydantic.
+2. The `parse_book_recommendation` function serves as an Output Parser, transforming the raw response into a structured `Book` instance.
+3. We use the `output_parser` parameter in the `@anthropic.call` decorator to specify our parser.
+4. The `recommend_book` function now returns a `Book` object instead of a raw response.
 
 ### Type Safety and Proper Hints
 
-When using output parsers with Mirascope's call decorator, you benefit from accurate type hints that reflect the parser's output type.
+When using output parsers with Mirascope's call decorator, you benefit from accurate type hints that reflect the parser's output type. This enhances code reliability and helps catch potential type-related errors early in the development process.
+
+For example, in the code above, your IDE will recognize `book` as a `Book` instance, providing appropriate autocompletion and type checking for `book.title` and `book.author`.
+
+## Comparison with Official SDK
```

Comment:
this isn't a 1:1 comparison because it doesn't use the pydantic `Book` model even though it could